src/hooks/progress/use-tutorial-progress-mutations/index.ts

```diff
@@ -5,7 +5,7 @@ import {
 	createTutorialProgress as createTutorialProgressApi,
 	updateTutorialProgress as updateTutorialProgressApi,
 } from 'lib/learn-client/api/progress'
-import { progressPercentToLabel } from 'lib/learn-client/api/progress/formatting'
```

Comment:
There are many changes in this PR that rename progress `label` to progress `status`. Apologies for the noise!